tests/test_tti.py

```diff
@@ -67,8 +67,8 @@ def test_tti(dimensions, space_order):
     t0 = 0.0
     tn = 350.0
     nt = int(1+(tn-t0)/dt)
-    last = (nt - 1) % 3
-    indlast = [(last + 1) % 3, (last+2) % 3, last % 3]
+    last = (nt) % 3
+    indlast = [(last + 2) % 3, (last+1) % 3, last % 3]
 
```

Comment:
Just computing in what order is the the wavefield after computation (cyclic indices)